packages/webdriverio/src/commands/browser/url.ts

```diff
@@ -44,5 +44,5 @@ export default function url (path) {
         path = nodeUrl.resolve(this.options.baseUrl, path)
     }
 
-    return this.navigateTo(validateUrl(path))
+    return this.navigateTo(validateUrl(path, null))
```

Comment:
we could consider changing `this.navigateTo` to accept `undefined` as the second parameter, then we could leave this as it was

Reply:
the parameter is part of `validateUrl` which needs to get typed first